src/controller/CHIPDeviceController.h

```diff
@@ -715,7 +714,9 @@ class DLL_EXPORT DeviceCommissioner : public DeviceController,
     /* Callback when the previously sent CSR request results in failure */
     static void OnCSRFailureResponse(void * context, CHIP_ERROR error);
 
-    void ExtendArmFailSafeForFailedDeviceAttestation(Credentials::AttestationVerificationResult result);
+    void ClearAttestationDeviceInfo();
```

Comment:
This function doesn't seem to actually exist.

Reply:
The implementation was removed when we changed to UniquePtr. I'll also remove this declaration.